README.md

```diff
@@ -119,6 +119,33 @@ warning The case-insensitive file ..\cmc-citizen-frontend\node_modules\nyc\node_
 warning The case-insensitive file ..\cmc-citizen-frontend\node_modules\nyc\node_modules\ansi-regex shouldn't be copied twice in one bulk copy
 ```
 
+## Preview Environment
+
+We get a fully functional environment in Azure Kubernetes (AKS) per pull request. For more
+info see: [K8S README](/src/kubernetes/README.md)
+
+Example URL using PR number from GitHub: `https://cmc-citizen-frontend-pr-XXX.service.core-compute-preview.internal/dashboard`
```

Comment:
only for frontend - does a claimstore change have any way to rub on a frontend pulled from aat? it didn't in preview but would be good to clarify here either way

Reply:
I see what you mean - does this read better?